web/src/components/NavigationBarScrollContainer.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { ReactElement, ReactNode, RefObject, useEffect, useState } from 'react'
```

Comment:
In which file was this scroll behavior implemented before? Do we have it duplicated now?

Reply:
there was only the standard scrolling of a div container via css `overflow-x: auto` no additional logic was implemented before for web